src/wp-includes/kses.php

```diff
@@ -2645,15 +2645,18 @@ function _wp_add_global_attributes( $value ) {
 		'aria-describedby' => true,
 		'aria-details'     => true,
 		'aria-expanded'    => true,
+		'aria-hidden'      => true,
 		'aria-label'       => true,
 		'aria-labelledby'  => true,
-		'aria-hidden'      => true,
+		'aria-live'        => true,
 		'class'            => true,
 		'data-*'           => true,
 		'dir'              => true,
+		'hidden'           => true,
 		'id'               => true,
 		'lang'             => true,
 		'style'            => true,
+		'tabindex'         => true,
```

Comment:
I'm not really thrilled about adding tabindex as a globally allowed attribute in KSES. It might be worth looking at the specific cases where this is used and applying it using JS, instead.